config/application.rb

```diff
@@ -10,6 +10,7 @@ module TestApp
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.1
+    config.time_zone = 'New Delhi'
```

Comment:
in order to understand the diff between now and current, change this timezone to something other than India Time zone

Reply:
Yes sir changing timezone.
Time.now.zone: IST
Time.current.zone: EST

Time.now -> shows time in our local zone.
Time.current -> shows time in zone set in rails configuration